### PR TITLE
v4.3.1: add release notes

### PR DESCRIPTION
**Pull request by KanjiMonster**
Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>

### CODE REVIEW

File: release-notes.txt [hunk 1]
```diff
@@ -1,186 +1,45 @@
...
-- Accton AS4610: Added support for the AS4610-54T (B) model with reversed
-  airflow. Special thanks to Ralf Kundel for helping with testing the
-  compatibility.
+4.3.1 Release Notes
+Date: 2021-11-30
 
 ! Changed behaviour
-- Accton AS4610: Since we now respect the Bootloader's passed memory limits,
-  the available memory is reduced by 16 MiB.
+- Certain utilities like client_drivshell will now require root privileges
```
Comment (jklare) [inline release-notes.txt:6]: we might want to highlight here the severeness of what was fixed

**Comment by jklare** [comment]
lgtm, needs some minor updates to describe the changed behaviour in more detail

**Comment by KanjiMonster** [comment]
Yeah, plan is to update it when the release images are ready (also in case we need to postpone the release and update the date)

File: release-notes.txt [hunk 2]
```diff
@@ -1,186 +1,51 @@
...
-- Accton AS4610: Fixed the second CPU sometimes not coming up.
-- Accton AS4610: Properly respect U-Boot's passed memory limits.
-- Accton AS4610: 1G LEDs blink for traffic again.
+- Validate stp arguments in ofdpa-grpc instead of passing them on as is
+
+! Security fixes
+- Fixed an issue where unauthenticated users could inject shell commands
+  executed with root privileges over the network
+- Fixed an issue where local users could inject shell commands executed
+   with root privileges
```
Comment (jklare) [inline release-notes.txt:15]:
```suggestion
- Fixed issues where users could inject shell commands either locally or unauthenticated over the network 
   being executed with root privileges
```

**Review by jklare — CHANGES_REQUESTED** [review]

File: release-notes.txt (hunk 2 above)
Comment (KanjiMonster) [reply release-notes.txt:15]: Not sure if it makes sense merging these two distinct issues. If we do, I would at least move the executed with root privileges to the shell commands, and maybe s/users/actors/.
e.g.
```suggestion
- Fixed issues allowing actors to inject shell commands executed with root privileges locally and unauthenticated over the network.
```

Comment (jklare) [reply release-notes.txt:15]: sounds good

**Review by jklare — APPROVED** [review]
lgtm

**Pull request merged**